React/tic-tac/src/app/components/withIsLoading/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import style from './style.scss';
+
+function LoaderHOC(WrappedComponent) {
+  return function wrapper(props) {
+   return props.userLoading ? (
+      <div className={ style.skcubegrid}>
+        <div className={` ${style.skcube} ${style.skcube1} `} />
```

Comment:
Delete spaces between clasess in all this divs
```suggestion
        <div className={`${style.skcube} ${style.skcube1}`} />
```